src/main/java/com/compilercharisma/chameleonbusinessstudio/client/VendiaClient.java

```diff
@@ -1,163 +1,33 @@
-                    _UserItems {
-                      _id
-                      email
-                      displayName
-                      role
-                    }
-                  }
-                }
-                """.formatted(email);
-        return getEntity(query, UserResponse.class)
-                .map(r -> !r.getUsers().isEmpty());
-    }
-
-    /**
-     * Edits a users info in Vendia Share
-     * @param user the user whose info will be edited in Vendia
-     * @return {@link User}
-     */
-    public Mono<User> updateUser(User user){
-        String updateUserMutation = """
-                mutation {
-                   update_User(
-                     id: "%s"
-                     input: {displayName: "%s", role: %s, email: "%s"}
-                   ) {
-                     result {
-                       firstName
-                       email
-                       lastName
-                       role
-                     }
-                   }
-                 }
-                """.formatted(user.get_id(), user.getDisplayName(), user.getRole(), user.getEmail());
-        return sendEntity(updateUserMutation, "update_User", User.class);
-    }
-
-    /**
-     * This deletes a user from the Vendia database
-     * Needs to go from user to _id ideally.
-     *
-     * @param user The user to be deleted.
-     * @return {@link UserResponse}
-     */
-    public Mono<UserResponse> deleteUser(User user) {
-        String deleteUserMutation = """
-                mutation {
-                  remove_User(id: "%s") {
-                    transaction {
-                      _id
-                    }
-                  }
-                }""".formatted(user.get_id());
-        return getEntity(deleteUserMutation, UserResponse.class);
-    }
-
-    private Mono<UserResponse> getUserId(String email) {
-        var query = """
-                query {
-                  list_UserItems(filter: {email: {eq: "%s"}}) {
-                    _UserItems {
-                      _id
-                      email
-                      displayName
-                      role
-                    }
-                  }
-                }
-                """.formatted(email);
-        return getEntity(query, UserResponse.class);
-    }
-
-    private <T> Mono<T> getEntity(String graphQlQuery, final Class<T> responseClass) {
-        return httpGraphQlClient
-                .document(graphQlQuery)
-                .retrieve("list_UserItems")
-                .toEntity(responseClass)
-                .doOnSubscribe(s -> log.info("Executing query to Vendia..."))
-                .doOnNext(n -> log.info("Finished executing query to Vendia"));
-    }
-
-    public <T> Mono<T> sendEntity(final String graphQlQuery, final String queryName, final Class<T> responseClass) {
+    public <T> Mono<T> executeQuery(String graphQlQuery, final String path, final Class<T> responseClass) {
```

Comment:
So instead of having methods for individual actions like getting user's id, emails, deleting them, etc, we're going to have this one method: `executeQuery()` to handle all those possible queries?

Reply:
Ah actually, I get the impression you're just removing test code?